src/data-channel-wrapper.cpp

```diff
@@ -350,6 +362,8 @@ void DataChannelWrapper::onClosed(const Napi::CallbackInfo &info)
                 // This will run in main thread and needs to construct the
                 // arguments for the call
                 args = {};
+            },[this]{
+                doCleanup();
```

Comment:
If I'm not mistaken, `doCleanup()` won't be called if the user did not set a closed callback.